homeassistant/components/conversation.py

```diff
@@ -74,6 +96,10 @@ def process(service):
             logger.error('Got unsupported command %s from text %s',
                          command, text)
 
+    # Use action if configured, otherwise use fuzzy "turn on/off" matching
+    process = process_action if action else process_fuzzy
```

Comment:
this makes very little sense to me. Why would one approach disable the other? Our actions are fairly limited and so you won't be able to do all text processing in it.

If you want actions in conversation, you will have to add support for sentences like `activate <>` and then see if you can find a script with that name that you can turn on.

Or, since scripts use the same service as switches and lights etc, you can already say `turn <script name> on` and it will already do the same.

If you care about the text being passed on as a variable, then _that_ is what you should implement.